src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractSyncNamespaceCodeRefactoringProvider.cs

```diff
@@ -22,6 +22,12 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             var textSpan = context.Span;
             var cancellationToken = context.CancellationToken;
 
+            var workspace = document.Project.Solution.Workspace;
+            if (workspace.Kind == WorkspaceKind.MiscellaneousFiles || document.IsGeneratedCode(cancellationToken))
```

Comment:
Can we add a test for this as well?

Reply:
I will try to add one

Reply:
Will do this in a follow up PR, since this change doesn't actually fix customer's issue.